screensaver/README.md

```diff
@@ -3,15 +3,15 @@
 Given a list of directories containing photos of different trips or themes,
 this script allows you to randomly select one and upload to a location from which a 
 photo rendering app reads from, e.g. your laptop's screensaver application or 
-a similar one in a Raspberry Pi
+a similar one in a RPi
 
 # Usage
 
 The script `screensaver.py` requires `python 3.7` with `numpy` and setting 3 environment variables:
 
-1. `SCREENSAVER_INPUT_PATH` (the path to where the photos directories are located)
-2. `SCREENSAVER_OUTPUT_PATH` (the path to where to upload the photos so the photo rendering app can read from)
-3. `SCREENSAVER_RSYNC_PORT` (the port over which ssh is being used, usually 22)
+1. `SCREENSAVER_INPUT_PATH` (The path to where the photos directories are located. Needs to end with `/`!)
+2. `SCREENSAVER_OUTPUT_PATH` (The path to where to upload the photos. Should not end with `/`)
```

Comment:
I see you somehow built these requirements into the code here https://github.com/tsando/photo-manager/blob/565be0916dca43db37b020f2e493e364c1c51630/screensaver/screensaver.py#L17-L21 so do we still need to ask users to how to end the input and output paths?

Reply:
Yes, I would say so, in case they miss it. But I will adjust the language.